firebase-firestore/src/test/java/com/google/firebase/firestore/LoadBundleTaskTest.java

```diff
@@ -61,7 +61,20 @@
 
   @Test
   public void testImplementsAllTaskInterface() {
+    // Check if the internal gms Hide annotation is accessible.
+    Class hideClazz = null;
+    try {
+      hideClazz = Class.forName("com.google.android.gms.common.internal.Hide");
```

Comment:
If we found this class, should we just exit the test?

Reply:
Why? The presence of this class will only indicate we are running on internal platform, we still need to implement all methods, isn't it?

Reply:
Please disregard. I though the statements in line 74 were or-ed together. LGTM.